### PR TITLE
Refactor scaling and add smoothed 5m training loop

### PR DESCRIPTION
**Pull request by markmonk123**
## Summary
- Disable all data normalizers across forecasting modules
- Add 5-minute smoothed data loop with 30k point retention
- Log half-hour forecasts and incorporate previous prediction into retraining
- Automatically drop unexpected normalized fields from incoming and training data

## Testing
- `python test_runner.py` *(fails: KeyboardInterrupt during data fetch)*
- `python test_month.py`

------
https://chatgpt.com/codex/tasks/task_e_68c1df92e6e8832cb0cb792b7d94e61b

### CODE REVIEW

File: continuous_training.py [hunk 1]
```diff
@@ -145,25 +178,56 @@ def _model_training_loop(self, model_id):
...
+                    if self.last_prediction is not None and len(new_data) > 30000:
+                        new_data = new_data.tail(30000)
+                    if self.last_prediction is not None and len(new_data) >= 29999:
+                        actual_price = new_data.iloc[-1]['price']
+                        actual_time = pd.to_datetime(new_data.iloc[-1]['date'])
+                        predicted_time = actual_time + timedelta(minutes=5)
+                        historical = new_data.iloc[-(29998+1):-1]
+                        head_rows = pd.DataFrame({
+                            'date': [predicted_time, actual_time],
+                            'price': [self.last_prediction, actual_price]
```
[the remark this answers is not in the document]
Comment (markmonk123) [reply continuous_training.py]:
```suggestion
                            'date': [predicted_time, actual_time],
                            'price': [self.last_prediction, actual_price]
```
